src/effects/backends/builtin/metronomeclick.cpp

```diff
@@ -4387,9 +4387,10 @@ std::span<const CSAMPLE> clickForSampleRate(mixxx::audio::SampleRate size) {
         return kClick44100;
     case 48000:
         return kClick48000;
-    case 98000:
+    case 96000:
```

Comment:
whoops, good catch. We should think on how we can avoid that. Maybe constrain the `SampleRate` type to only accept select rates?